lib/httparty/request.rb

```diff
@@ -276,7 +276,7 @@ def handle_response(body, &block)
           unless options[:maintain_method_across_redirects] && options[:resend_on_redirect]
             self.http_method = Net::HTTP::Get
           end
-        else
+        elsif last_response.code.to_s != '307' && last_response.code.to_s != '308'
```

Comment:
Why the conversion to string? Seems like that just allocates more strings both for the "307" and the code.to_s when integer would be fine.